Makefile

```diff
@@ -12,8 +12,9 @@ run: build ## Build & Run the demucs spliting the tracks placed in the input fol
 		-v $(current-dir)input:/data/input \
 		-v $(current-dir)output:/data/output \
 		-v $(current-dir)models:/data/models \
+		-e TORCH_HOME=/data/models \
```

Comment:
I think we could set the environment variable in the `Dockerfile` after [this line](https://github.com/xserrat/docker-facebook-demucs/blob/main/Dockerfile#L3) because the volume is part of the image, not the running container. What do you think?

Reply:
Yes it makes sense. I will change it.